### PR TITLE
8636 user group removal

### PR DESCRIPTION
**Pull request by smoyth**
Mostly fixes up index links and adds a feature spec. The feature itself was no longer broken it seems.

### CODE REVIEW

File: app/helpers/questions_helper.rb [hunk 1]
```diff
@@ -4,32 +4,22 @@
...
     links = []
-
-    # links for form mode
+    # Links for when adding questions to form
     if params[:controller] == "forms"
-      # add the 'add questions to form' link if there are some questions
       unless @questions.empty?
-        links << batch_op_link(name: t("form.add_selected"), path: add_questions_form_path(@form))
+        links << batch_op_link(name: t("action_links.models.question.add_to_form"),
+                               path: add_questions_form_path(@form))
```
[the remark this answers is not in the document]
Comment (smoyth) [reply app/helpers/questions_helper.rb:11]: Legacy

File: app/assets/stylesheets/local/_questions.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@ form.question_form {
 .questions_index,
 .forms_choose_questions,
 .forms_edit {
-  #index_table {
+  .index_table {
```
[the remark this answers is not in the document]
Comment (smoyth) [reply app/assets/stylesheets/local/_questions.scss:8]: Legacy

File: app/assets/stylesheets/global/_index_tables.scss [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
+  }
+
+  .link-divider {
+    display: inline-block;
+    padding: 0 5px;
+    color: $light-border-color;
+  }
+}
+
+.index_table {
```
[the remark this answers is not in the document]
Comment (smoyth) [reply app/assets/stylesheets/global/_index_tables.scss:25]: Legacy

**Review by cooperka — COMMENTED** [review]
Nice changes, generally LGTM, though there are now 2 kinds of link colors (both screenshots show both colors for various links, `$accent-color` and `$darker-accent-color`):

Screen Shot 2020-12-30 at 11 54 15
Screen Shot 2020-12-30 at 11 55 09

**Comment by cooperka** [comment]
Note: I rebased after the latest merge and resolved a minor conflict. Travis CI passed on your latest commit before rebase. We are now **out of Travis credits** for the month...

**Comment by smoyth** [comment]
Regarding two link colors, that was on purpose, to help differentiate the collection links from the batch links. Not ideal I know. Eventually the collection links should be elsewhere on the screen I think but baby steps!

**Pull request merged**